middleware/LoggingHandler.md

```diff
@@ -1,11 +1,21 @@
-# Logging Handler [In Progress]
+# Logging Handler [Draft 1]
 
 ## Objectives
 
-This component should allow a developer to provide their own logging listener and important information related to the request and response should be written to the listener.  Where the native HTTP client library has its own built in logging infrastructure, that should be preferred over a custom solution.
-
-## Requirements
+This feature is a middleware component that allows developers to plug in their own 3rd party logging platform to there application, and monitor items relating to the latency of the requests and responses of the handlers in the middleware pipeline. These items will be provided to a thrid party logging framework. This feature also allows you to configure a mechanism to track request that go over a specified time and provide that information to 3rd party apropriate log level. 
 
+## Requirements 
+ <ul>
+ <li>The logger will allow users to trace various spans that occur during the authentication and the call to graph service but will have the potential to trace other operations and requests.</li> 
+ <li>Each span is identified by it's name, span ID, start and end time (latency), the trace id it's assoicated with, as well as other           fields.</li>
```

Comment:
SpanID SHOULD be the as the client-request-id if it is set by the client.  We may want to consider using the request-id returned in a response to specify the SpanID. We should consider what happens when we don't get a response - the request times out.